src/Security/ConnectAuthenticator.php

```diff
@@ -140,7 +140,9 @@ public function authenticate(Request $request): PassportInterface
             throw new AuthenticationServiceException('The user provider must return a UserInterface object.');
         }
 
-        $passport = new SelfValidatingPassport(new UserBadge($localUser), []);
+        $passport = new SelfValidatingPassport(new UserBadge($localUser->getUuid(), function (string $symfonyConnectUuid) {
```

Comment:
Isn't it the default behavior?

Reply:
I'm not entirely sure. But, at least my own Symfony app, only works if I make this change.

Maybe @wouterj or @chalasr can clarify if I'm doing something wrong. Thanks!

Reply:
Yes, it is. Have you configured the correct user provider, @javiereguiluz ?

Reply:
I've made further changes to remove this code, which indeed is the default behavior. Thanks!